src/app/features/search/search.component.ts

```diff
@@ -19,11 +19,11 @@ export class SearchComponent {
 
   public searchTechRecords(searchTerm: string) {
     const searchErrorMessage = 'You must provide a vehicle registration mark, trailer ID or vehicle identification number.';
-
+    const trimmedSearchTerm = searchTerm.trim();
```

Comment:
we could just do 
searchTerm = searchTerm.trim();
as we don't need the original, and would fix the bug below